cypress/integration/navigateHomeAndContactsPages.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('navigateHomeAndContactsPages', function() {
+
+    const operations = [
+        {
+            // Estonian
+            homePage: 'https://ut2019.voog.com/et',
+            contactPage: 'https://ut2019.voog.com/contact-1'
+
+        },
+        {
+            // English
+            homePage: 'https://ut2019.voog.com/en',
+            contactPage: 'https://ut2019.voog.com/contact'
+        }
+    ];
+
+    operations.forEach((operation) => {
+        it('Navigated successfully', function() {
+            cy.visit(operation.homePage);
+
+            cy.visit(operation.contactPage);
+            cy.url().should('eq', operation.contactPage);
```

Comment:
It is good that you assert the page url is correct after opening it with cy.url().should('eq', operation.homePage); but you should also check some element's content inside the page to really make sure a correct page is rendered not just that the url is what you expected it would be.